### DISCUSSION TITLE
Change accordian click box to allow copy/pasting from event

### DISCUSSION THREAD

**Discussion by ranok** [thinkst/canarytokens#252] [Feature Requests]
In the token history (Web Bug if it matters) there is a list of collapsed incidents. When I click on one to open it, the entire box is an onClick to close it, so I cannot copy information out of the box.

It would be great if only clicking on the date/time (header) for the incident minimizes the incident so I can copy and paste information out of the incident.

**Comment by roberto-aldera** [comment]
Fixed in new UI
